near/omni-bridge/src/storage.rs

```diff
@@ -210,10 +210,7 @@ impl Contract {
             sender: OmniAddress::Near(max_account_id.clone()),
             msg: msg.unwrap_or_default(),
             destination_nonce: 0,
-            origin_transfer_id: Some(TransferId {
-                origin_chain: ChainKind::Near,
-                origin_nonce: 0,
-            }),
+            origin_transfer_id: Some(UnifiedTransferId::Utxo("a".repeat(64))),
```

Comment:
75 = 64 + 1 + 10 = tx_id@vout

Reply:
Wonder if there's any benefit making UTXOTransferId typed with separate tx_id and vout fields